pkgs/development/python-modules/filelock/3.2.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, buildPythonPackage, fetchPypi }:
+
+buildPythonPackage rec {
+  pname = "filelock";
+  version = "3.2.0";
```

Comment:
Any reason to not build version 3.2.1 instead? https://github.com/tox-dev/py-filelock/releases/tag/3.2.1

Python 2 support was dropped in version 3.3.0: https://github.com/tox-dev/py-filelock/releases/tag/3.3.0

Reply:
That's an oversight on my side. I'll fix it right now.

Reply:
Please also fix the commit message:

```
python2Modules.filelock: init at version 3.2.1
```